data/json/items/ammo/300blk.json

```
@@ -37,7 +37,7 @@
     "id": "bp_300blk",
     "copy-from": "300blk",
     "type": "AMMO",
-    "name": { "str": ".300 AAC Blackout, black powder" },
+    "name": { "str": ".300 AAC Blackout, black powder", "str_pl": ".300 AAC Blackouts, black powder" },
```

```suggestion
    "name": { "str_sp": ".300 AAC Blackout, black powder" },
```
Or is there any difference that makes `Blackouts` acceptable here?

---

I looked up the usage of the term - people do write (online, at least) `Blackouts` when they're talking about more than one `AAC Blackout` round. (They don't write "FMJs", however, and "Magnums" appears to be only used in talking about guns that fire them.)

---

In other words, "Blackout" is a noun, not an adjective (modifying "round" or "rounds").